pkg/storage/repodb/common.go

```diff
@@ -0,0 +1,64 @@
+package repodb
```

Comment:
This is somewhat related to storage but not entirely?
so not sure it should live under pkg/storage/...

Reply:
It is only in storage because it deals with a lot of information contained in manifest and config structs, but it could be moved to pkg/repodb, or pkg/meta/repodb.